Migration/jcl/groovy/JCLtoDBB.groovy

```diff
@@ -467,7 +467,16 @@ def convertAllocationToDD(def concat)
 			dd.'pass' = true
 		else
 		{
-			options << "${concat.dispnor}".toLowerCase()
+			def dispnorValue = "${concat.dispnor}".toLowerCase()
+            //* Updated to use catalog/uncatalog instead for catlg/uncatlg for BPXWDYN utility
```

Comment:
Rather than having the _update comment_, lets indicate, that we 
map the disposition `catlg/uncatlg` to `catalog/uncatalog` to align to BPXWDYN utility options

Reply:
Updated the changes and created a new pull request - https://github.com/IBM/dbb/pull/126.  This pull request seems to be from my personal github account.